src/Solr/Query/SearchQueryBuilder.php

```diff
@@ -59,8 +59,16 @@
      * @param int $storeId
      * @param EventDispatcher $eventDispatcher
      */
-    public function __construct(SearchString $searchString, FuzzyConfig $fuzzyConfig, ResultsConfig $resultsConfig, AttributeRepository $attributeRepository, Pagination $pagination, ParamsBuilder $paramsBuilder, $storeId, EventDispatcher $eventDispatcher)
-    {
+    public function __construct(
+        SearchString $searchString,
+        FuzzyConfig $fuzzyConfig,
+        ResultsConfig $resultsConfig,
+        AttributeRepository $attributeRepository,
+        Pagination $pagination,
+        ParamsBuilder $paramsBuilder,
```

Comment:
should be `SearchParamsBuilder` now, because methods not in the interface are used (which is okay in this case, as discussed)

Reply:
Okay, I changed this. See also https://github.com/integer-net/solr-pro/pull/7 as I had to do changes to the pro library too due to changed interface.

Reply:
I can't change it to `SearchParamsBuilder` easily. It crashes with the Autosuggest.